submissions/ses3332002/memory-game/script.js

```diff
@@ -0,0 +1,156 @@
+"use strict";
+  
+  const doc = document.body;
+  const themeButtons = document.querySelectorAll(".theme_selector__button");
+  const sizeSelector = document.querySelector(".size_selector");
+  const cardThemeSelector = document.querySelectorAll('input[name="option"]');
+  const startButton = document.querySelector(".start_button");
+  const gameArea = document.querySelector(".game_area");
+  let numbersArray;
+  let cardThemeSelected;
+  let cardSelected;
+  let cardsArray;
+  let matchedCounter;
+  
+  startButton.addEventListener("click", startGame);
+  doc.classList.add("theme1");
+
+  themeButtons.forEach((el,i) => {
+    el.classList.add("theme"+(i+1));
+    el.addEventListener("click", function() {
+      setupTheme(i);
+    });
+  });
+
+  function setupTheme(i) {
+    doc.classList.remove(doc.classList[0]);
+    doc.classList.add("theme"+(i+1));
+  };
+  
+  function startGame() {
+    cardThemeSelected = document.querySelector('input[name="option"]:checked');
+    cardThemeSelector.forEach((item) => {
+      item.disabled = true;
+      item.nextElementSibling.classList.add("form_item-desabled");
+      item.nextElementSibling.style.pointerEvents = "none";
+    });
+    sizeSelector.disabled = true;
+    sizeSelector.classList.add("form_item-desabled");
+    startButton.disabled = true;
+    startButton.classList.add("form_item-desabled");
+    startButton.style.pointerEvents = "none";
```

Comment:
You do some operations on certain elements. You can add one class to these elements and use the method `classList.toggle()` to attach and cancel a certain state to them.

```
const importantIems = document.querySelectorAll('.importantIems');
  function changeImportantItems() {
    importantIems.forEach(function(item) {
      item.classList.toggle('form_item-desabled');
    })
  }
```

Reply:
I can remake it.

Reply:
Well...
I need to search and select this "important items" anyway to get their values for preparing my gamefield. So I already have this items in my variables. 
With your method I would need to give additional class to this items only for searching them and instead of eight lines of disabling items in "my" way I would need to use seven lines (with "item.disabled = true;") in "your" way. 
Is it optimally?

Reply:
By calling this function, you simplify the `resetWindow()` function also.
Because in it you change properties and remove classes.

Reply:
I remade controls switching